packages/main/src/components/AnalyticalTable/TableBody/RowSubComponent.tsx

```diff
@@ -1,6 +1,20 @@
+import { ThemingParameters } from '@ui5/webcomponents-react-base/dist/ThemingParameters';
 import React, { useEffect, useRef } from 'react';
+import { createUseStyles } from 'react-jss';
 import { VirtualItem } from 'react-virtual';
 
+const styles = {
+  subcomponent: {
+    position: 'absolute',
+    width: '100%',
+    '&:focus': {
+      outlineOffset: '-2px',
```

Comment:
Why do we use -2px here?
Would this work as well?
```suggestion
      outlineOffset: 'calc(-1 * ${ThemingParameters.sapContent_FocusWidth})',
```

Reply:
I'm not really sure why :smiley:

As this is working and even looks better and is more compliant, I will change it here and in the Cell styles.